Rules/AvoidAlias.cs

```diff
@@ -126,25 +126,29 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
                         fileName,
                         commandName,
                         suggestedCorrections: GetCorrectionExtent(cmdAst, cmdletNameIfCommandNameWasAlias));
+                    // do not continue the search, but go to the next command
+                    continue;
+                }
+
+                // If we find match of any kind, do not continue with the Get-{commandname} check
+                if ( Helper.Instance.GetCommandInfo(commandName) != null ) {
+                    continue;
                 }
 
-                var isNativeCommand = Helper.Instance.GetCommandInfo(commandName, CommandTypes.Application | CommandTypes.ExternalScript) != null;
-                if (!isNativeCommand)
+                var commdNameWithGetPrefix = $"Get-{commandName}";
+                var cmdletNameIfCommandWasMissingGetPrefix = Helper.Instance.GetCommandInfo($"Get-{commandName}");

```

Comment:
We should reuse the `commdNameWithGetPrefix` variable defined on the line above here:
```suggestion
                var cmdletNameIfCommandWasMissingGetPrefix = Helper.Instance.GetCommandInfo(commdNameWithGetPrefix);
```